packages/client/tests/functional/_utils/config-builder.ts

```diff
@@ -0,0 +1,41 @@
+import path from 'node:path'
+
+import { defaultTestConfig, type PrismaConfigInternal } from '@prisma/config'
+
+import { getTestSuiteSchemaPath, type NamedTestSuiteConfig, type TestSuiteMeta } from './getTestSuiteInfo'
+import type { DatasourceInfo } from './setupTestSuiteEnv'
+
+export function buildPrismaConfig({
+  suiteMeta,
+  suiteConfig,
+  datasourceInfo,
+}: {
+  suiteMeta: TestSuiteMeta
+  suiteConfig: NamedTestSuiteConfig
+  datasourceInfo: DatasourceInfo
+}) {
+  const schemaPath = getTestSuiteSchemaPath({ suiteMeta, suiteConfig })
+  return buildPrismaConfigInternal(schemaPath, datasourceInfo.databaseUrl)
+}
+
+function buildPrismaConfigInternal(schemaPath: string, databaseUrl: string): PrismaConfigInternal {
+  return {
+    ...defaultTestConfig(),
+    engine: 'classic',
+    schema: schemaPath,
+    datasource: {
+      url: absolutizeSqliteUrl({ schemaPath, databaseUrl }),
+    },
+  }
+}
+
+const SQLITE_PROTOCOL = 'file:'
```

Comment:
```suggestion
const FILE_PROTOCOL = 'file:'
```
right now it'll only happen for sqlite, but there's nothing sqlite specific about this code, so I'd just call it `FILE_PROTOCOL`